examples.sty

```diff
@@ -244,3 +252,154 @@
+  \anchor{south west}{\pgf@process{\outernortheast}\pgf@x=-.5\pgf@x\pgf@y=-.5\pgf@y}%
+  \anchor{east}{\pgf@process{\outernortheast}\pgf@y=0pt}%
+  \anchor{north east}{\pgf@process{\outernortheast}\pgf@x=.5\pgf@x\pgf@y=.5\pgf@y}%
+  \anchor{south east}{\pgf@process{\outernortheast}\pgf@x=.5\pgf@x\pgf@y=-.5\pgf@y}%
+  \anchorborder{%
+    \pgf@xa=\pgf@x%
+    \pgf@ya=\pgf@y%
+    \pgf@process{\outernortheast}%
+    \ifdim\pgf@xa>0pt%
+    \else%
+      \pgf@x=-\pgf@x%
+    \fi%
+    \ifdim\pgf@ya>0pt%
+    \else%
+      \pgf@y=-\pgf@y%
+    \fi%
+    \edef\pgf@marshal{%
+      \noexpand\pgfpointintersectionoflines
+      {\noexpand\pgfpointorigin}
+      {\noexpand\pgfqpoint{\the\pgf@xa}{\the\pgf@ya}}
+      {\noexpand\pgfqpoint{\the\pgf@x}{0pt}}
+      {\noexpand\pgfqpoint{0pt}{\the\pgf@y}}%
+    }%
+    \pgf@process{\pgf@marshal}%
+  }%
+
+  %
+  % Background path
+  %
+  \backgroundpath{
+    \pgf@process{\outernortheast}%
+%    \pgfsetroundcap
+    \pgf@xc=\pgf@x%
+    \pgf@yc=\pgf@y%
+	\pgfsetroundjoin
+	\pgfsetroundcap
+    \pgfmathsetlength{\pgf@xa}{\pgfkeysvalueof{/pgf/outer xsep}}%
+    \pgfmathsetlength{\pgf@ya}{\pgfkeysvalueof{/pgf/outer ysep}}%
+    \advance\pgf@xc by-1.414213\pgf@xa%
+    \advance\pgf@yc by-1.414213\pgf@ya%
+    \pgfpathmoveto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}{0pt}}}%
+    \pgfpathlineto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{0pt}{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}}}%
+     \pgfpathlineto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}{0pt}}}%
+    \pgfpathlineto {\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{0pt}{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}}}%
+    \pgfpathlineto{\pgfqpoint{\pgf@xc}{-\pgf@yc}}%
+    \pgfpathlineto{\pgfqpoint{-\pgf@xc}{-\pgf@yc}}%
+    \pgfpathlineto{\pgfqpoint{-\pgf@xc}{\pgf@yc}}%
+	\pgfpathclose
+    \pgf@process{\outernortheast}%
+    \pgfpathmoveto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}}}%
+    \pgfpathlineto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{0pt}{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}}}%
+    \pgfpathmoveto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}}}%
+    \pgfpathlineto{\pgfpointadd{\pgfqpoint{\pgf@xc}{\pgf@yc}}{\pgfpoint{-\pgfkeysvalueof{/tikz/document-corner}\pgf@xc}{0pt}}}%
+    %
+    \ifDocumentWithText
+    \foreach \y in {-.8,-.4,0} {
+		\pgfpathmoveto{\pgfpointadd{\pgfqpoint{-.65\pgf@xc}{0pt}}{\pgfpoint{0}{\y\pgf@xc}}}%
+		\pgfpathlineto{\pgfpointadd{\pgfpoint{.65\pgf@xc}{0pt}}{\pgfpoint{0}{\y\pgf@xc}}}%
+    }
+   \fi
+  }%
+}%
+
+\tikzset{
+	code-document/.style={document,fill=white,very thick,minimum height=45pt,draw=black,node contents=\ttfamily</>},
```

Comment:
Perhaps we can change the font size in `\small_icon:n` and `\large_icon:nn` to make the contents (`</>`) scale better. It's currently quite tiny in the large icon.

Reply:
in increased it a bit. general issue here is, that we now scale the icon with the font. I added support for reducing the `inner~sep` to the shape. Guess its okay now, but please have a look.